tests/unit/fern/test_fern_docs.py

```
@@ -64,6 +101,7 @@ def test_fern_docs_dev_starts() -> None:
     port = _get_free_port()
     proc = subprocess.Popen(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)